ui/src/styles/index.scss

```diff
@@ -178,8 +185,23 @@
   margin-right: 16px;
 }
 
-.ant-layout .ant-layout-header {
-  line-height: inherit;
+.ant-layout {
+  .responsive-header {
+    line-height: inherit;
+    border-bottom: 1px solid #eaecf0;
```

Comment:
This should be defined and used as a new variable, e.g. `$divider-color: #eaecf0;`.

Also noticing another hardcoded color value on line 168 - it should instead be `color: $error-color;`.